pkg/controller/volume/persistentvolume/framework_test.go

```diff
@@ -595,22 +599,62 @@ func newTestController(kubeClient clientset.Interface, volumeSource, claimSource
 	if classSource == nil {
 		classSource = fcache.NewFakeControllerSource()
 	}
-
+	// Pass in shared informers as usual but do not start them. Instead,
```

Comment:
k it wasn't as big a change as I thought it would be. still need to fix the tests etc. but this part is done I think.